tests/system/action/motion_comment/test_update.py

```diff
@@ -15,13 +34,52 @@ def test_update_correct(self) -> None:
         assert model.get("comment") == "comment_Xcdfgee"
 
     def test_update_wrong_id(self) -> None:
-        self.create_model(
-            "motion_comment/111",
-            {"comment": "comment_srtgb123"},
+        self.set_models(
+            {
+                "meeting/1": {},
+                "motion_comment/111": {
+                    "comment": "comment_srtgb123",
+                    "meeting_id": 1,
+                    "section_id": 78,
+                },
+                "motion_comment_section/78": {"meeting_id": 1, "write_group_ids": [3]},
+            }
         )
         response = self.request(
             "motion_comment.update", {"id": 112, "comment": "comment_Xcdfgee"}
         )
         self.assert_status_code(response, 400)
         model = self.get_model("motion_comment/111")
         assert model.get("comment") == "comment_srtgb123"
+
+    def test_update_no_permission(self) -> None:
+        self.base_permission_test(
+            self.permission_test_model,
+            "motion_comment.update",
+            {"id": 111, "comment": "comment_Xcdfgee"},
+        )
+
+    def test_update_permission(self) -> None:
+        self.base_permission_test(
+            self.permission_test_model,
+            "motion_comment.update",
+            {"id": 111, "comment": "comment_Xcdfgee"},
+            Permissions.Motion.CAN_SEE,
+        )
+
+    def test_update_no_permission_cause_write_group(self) -> None:
```

Comment:
same as above

Reply:
Update the default test to check this, too.